posthog/management/commands/migrate_replay_retention.py

```diff
@@ -14,31 +15,44 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         dry_run = options["dry_run"]
         batch_size = options["batch_size"]
-        teams_to_migrate = []
 
         try:
             self.stdout.write(
                 self.style.SUCCESS("Starting Replay retention migration" + (" (DRY RUN)" if dry_run else ""))
             )
 
-            for team in Team.objects.filter(session_recording_retention_period="legacy").only(
-                "id", "organization", "session_recording_retention_period"
-            ):
-                # NOTE: We use file export as a proxy to see if they are subbed to Recordings
-                is_paid = team.organization.is_feature_available(AvailableFeature.RECORDINGS_FILE_EXPORT)
-                team.session_recording_retention_period = "90d" if is_paid else "30d"
+            total_teams_migrated = 0
 
-                teams_to_migrate.append(team)
-                self.stdout.write(self.style.SUCCESS(f"{'Would migrate' if dry_run else 'Migrating'} team {team.id}"))
+            queryset = (
+                Team.objects.filter(session_recording_retention_period="legacy")
+                .order_by("id")
+                .only("id", "organization", "session_recording_retention_period")
```

Comment:
combination of `.only` here and specific field update below is :chefs-kiss.meme: